project.clj

```diff
@@ -16,7 +16,7 @@
                  [quil/processing-svg "3.2.2"]
                  [quil/jogl-all-fat "2.3.2"]
                  [quil/gluegen-rt-fat "2.3.2"]
-                 [quil/processing-js "1.6.3.0"]
+                 [quil/processing-js "1.6.4.1"]
```

Comment:
You should pull master branch. Currently 1.6.4.0 is used: https://github.com/quil/quil/blob/master/project.clj#L19

Also I need to push 1.6.4.1 to clojars then.